File: Swift/1-50/13.swift
if let input = readLine() {
    let inputs = input.components(separatedBy: " ")
    if let n = Int(inputs[0]), let k = Int(inputs[1]) {
        if n == 0 {
            print("1")
        } else {
            print(k + 1)
        }
    }
}

File: Php/1-50/13.php
<?php
$input = readline();
$inputs = explode(" ", $input);
$n = intval($inputs[0]);
$k = intval($inputs[1]);

if ($n == 0) {
    echo "1\n";
} else {
    echo ($k + 1) . "\n";
}
?>